packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -318,47 +197,66 @@ function check_lock_file()
     return true
 end
 
+-- Remove lock file
 function remove_lock_file()
-    execute("rm /etc/first_run")
+    utils.execute("rm /etc/first_run")
+end
+
+-- List downloaded lime-defaults
+function read_configs()
+    local tempFiles = fs.dir("/tmp/")
+    local result = {}
+    for file in tempFiles do
+        if (file ~= nil and file:sub(1, 12) == "lime-default") then
+            local ap = getAp(file)
+            table.insert(result, {
+                ap = string.gsub(ap, "\"", ""),
+                file = file
+            })
+        end
+    end
+    return result
 end
 
+-- Create new network
```

Comment:
the comment doesn't match the name of the function. please fix

Reply:
I change the comment line: _Apply configuration for a new network ( userd in ubus daemon)_